### ISSUE TITLE
[🧐 Evaluating] 🐞 Bug Report: Mega Menu - Dropdowns change size compared to previews

### ISSUE THREAD

**Issue by TelioHRV** [elementor/elementor#22749]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

The Mega Menu dropdowns change size when you preview or publish.

### Steps to reproduce

https://github.com/elementor/elementor/assets/136447101/1691fab6-51ff-4bac-bce8-02e972137da7

![Capture](https://github.com/elementor/elementor/assets/136447101/6ef47df2-b880-45fc-8491-62bb952b23eb)

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt


					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MariaDB Server v10.3.35
	PHP Version: 8.0.28
	PHP Memory Limit: 128M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 8M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.1.1
	Site URL: https://cottos.fr
	Home URL: https://cottos.fr
	WP Multisite: No
	Max Upload Size: 2 Mo
	Memory limit: 40M
	Max Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: fr-FR
	Timezone: 0
	Admin Email: [REDACTED]
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.7.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: fr_FR
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36

== Active Plugins ==
	Advanced Custom Fields
		Version: 6.1.6
		Author: WP Engine

	Analytics Cat Free
		Version: 1.1.2
		Author: Fatcat Apps

	Connect Polylang for Elementor
		Version: 2.4.2
		Author: Creame

	DynamicConditions
		Version: 1.6.2
		Author: RTO GmbH

	Elementor
		Version: 3.13.4
		Author: Elementor.com

	Elementor Pro
		Version: 3.13.2
		Author: Elementor.com

	HappyFiles Pro
		Version: 1.8.2
		Author: Codeer

	Header Footer Code Manager
		Version: 1.1.32
		Author: 99robots

	Polylang
		Version: 3.4.2
		Author: WP SYNTEX

	Redirection
		Version: 5.3.10
		Author: John Godley

	SVG Support
		Version: 2.5.5
		Author: Benbodhi

	W3 Total Cache
		Version: 2.3.2
		Author: BoldGrid

	WP Revisions Control
		Version: 1.4.3
		Author: Erick Hitter

	Yoast Duplicate Post
		Version: 4.5
		Author: Enrico Battocchi & Team Yoast

	Yoast SEO
		Version: 20.9
		Author: Team Yoast


== Utilisation des éléments ==
	
	archive : 1
		button : 4
		heading : 6
		icon-list : 1
		posts : 2
		text-editor : 3
	footer : 2
		heading : 2
		html : 1
		icon-list : 1
		image : 1
		social-icons : 1
		text-editor : 1
	header : 4
		button : 3
		container : 21
		divider : 3
		heading : 11
		image : 9
		mega-menu : 1
		nav-menu : 3
		polylang-language-switcher : 3
		text-editor : 3
	page : 6
		button : 11
		container : 3
		heading : 45
		image : 23
		image-carousel : 1
		text-editor : 25
	popup : 2
		html : 2
	section : 24
		button : 5
		divider : 3
		heading : 32
		icon-box : 16
		image : 13
		image-carousel : 3
		spacer : 1
		testimonial : 1
		text-editor : 20
	widget : 0
		button : 4
	wp-post : 7
		button : 5
		heading : 35
		image : 12
		spacer : 23
		text-editor : 52
		video : 1
	wp-page : 27
		button : 32
		container : 89
		heading : 152
		html : 9
		icon : 9
		icon-list : 2
		image : 95
		image-carousel : 2
		media-carousel : 2
		spacer : 16
		text-editor : 84
		video : 2
	single-post : 2
		button : 3
		divider : 3
		heading : 11
		image : 2
		table-of-contents : 1
		text-editor : 10
		theme-post-content : 2
		theme-post-title : 1
	error-404 : 1
		heading : 4
		icon-list : 1


== Réglages ==
	
	cpt_support: post, page, e-landing-page
	disable_color_schemes: yes
	disable_typography_schemes: yes
	allow_tracking: yes
	unfiltered_files_upload: 1


== Fonctionnalités ==
	Custom Fonts: 0
	Custom Icons: 0

== Intégrations ==
	


== Expériences d’Elementor ==
	Sortie DOM optimisée: Actif par défaut
	Amélioration du chargement des ressources: Actif
	Optimisation du chargement du CSS: Actif
	Icônes de police « inline »: Actif
	Améliorations de l’accessibilité: Actif par défaut
	Points d’arrêt personnalisés supplémentaires: Actif
	admin_menu_rearrangement: Inactif par défaut
	Conteneur Flexbox: Actif
	Mettre à niveau la bibliothèque Swiper: Inactif par défaut
	Grid Container: Inactif par défaut
	Par défaut vers le Nouveau constructeur de thème: Actif par défaut
	En-tête et pied de page du thème Hello: Inactif par défaut
	Editor Top Bar: Inactif par défaut
	Page d’atterrissage: Actif par défaut
	Éléments imbriqués: Actif
	Chargement différé des images d’arrière-plan: Inactif par défaut
	Transitions des pages: Actif par défaut
	Notes: Actif par défaut
	Boucle: Actif par défaut
	Envois de formulaire: Actif par défaut
	Accroche de défilement: Actif par défaut
	Menu: Actif


== Journal ==
	
JS: showing 10 of 10JS: 2022-11-08 15:20:59 [error X 1][https://cottos.fr/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.7.8:3:767341] Cannot read properties of undefined (reading \'toLowerCase\') 
JS: 2022-11-30 09:39:33 [error X 1][https://cottos.fr/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.7.8:3:692080] T.getContainer is not a function 
JS: 2022-12-21 16:41:40 [error X 4][https://cottos.fr/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.7.8:3:767848] Cannot read properties of undefined (reading \'localized\') 
JS: 2023-01-03 14:21:24 [error X 26][https://cottos.fr/wp-admin/post.php?post=12534&action=elementor#:1:1] Identifier \'$dcWidgetIdInput\' has already been declared 
JS: 2023-03-24 09:03:11 [error X 2][https://cottos.fr/wp-includes/js/jquery/jquery.min.js?ver=3.6.1:2:51824] Cannot read properties of undefined (reading \'replace\') 
JS: 2023-04-24 13:10:34 [error X 7][https://cottos.fr/wp-admin/post.php?post=14439&action=elementor#:1:1] Identifier '$dcWidgetIdInput' has already been declared 
JS: 2023-04-24 15:39:18 [error X 14][https://cottos.fr/wp-includes/js/jquery/jquery.min.js?ver=3.6.1:2:51824] Cannot read properties of undefined (reading 'replace') 
JS: 2023-04-25 09:35:33 [error X 12][https://cottos.fr/wp-content/plugins/elementor-pro/assets/js/frontend.min.js?ver=3.12.0:2:11958] Failed to execute 'observe' on 'IntersectionObserver': parameter 1 is not of type 'Element'. 
JS: 2023-05-10 09:44:37 [error X 2][https://cottos.fr/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.12.0:3:673293] this.model.isValidChild is not a function 
JS: 2023-06-09 14:32:00 [error X 14][https://cottos.fr/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.12.0:3:785745] Cannot read properties of undefined (reading 'localized') 

PHP: showing 10 of 10PHP: 2022-11-29 08:51:44 [notice X 3][/var/www/vhosts/cottos.fr/httpdocs/wp-content/plugins/elementor-pro/license/updater.php::84] Undefined index: new_version [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-01-03 09:59:47 [notice X 1][/var/www/vhosts/cottos.fr/httpdocs/wp-content/plugins/elementor/core/page-assets/data-managers/font-icon-svg/font-awesome.php::50] Trying to access array offset on value of type null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-01-03 10:46:37 [notice X 94][/var/www/vhosts/cottos.fr/httpdocs/wp-content/plugins/elementor-pro/modules/dynamic-tags/acf/tags/acf-text.php::33] Undefined offset: 1 [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-01-04 15:05:27 [notice X 320][/var/www/vhosts/cottos.fr/httpdocs/wp-content/plugins/elementor/includes/conditions.php::87] Undefined index: motion_fx_transform_x_anchor_point [array (
  'trace' => '
#0: /var/www/vhosts/cottos.fr/httpdocs/wp-content/plugins/elementor/includes/conditions.php(87): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /var/www/vhosts/cottos.fr/httpdocs/wp-content/plugins/elementor/includes/base/controls-stack.php(1383): Elementor\Conditions :: check()
#2: /var/www/vhosts/cottos.fr/httpdocs/wp-content/plugins/elementor/includes/base/controls-stack.php(1169): Elementor\Controls_Stack -> is_control_visible()
#3: /var/www/vhosts/cottos.fr/httpdocs/wp-content/plugins/elementor/includes/base/controls-stack.php(1210): Elementor\Controls_Stack -> get_active_settings()
#4: /var/www/vhosts/cottos.fr/httpdocs/wp-content/plugins/dynamicconditions/Public/DynamicConditionsPublic.php(118): Elementor\Controls_Stack -> get_settings_for_display()
',
)]
PHP: 2023-01-04 15:05:27 [notice X 320][/var/www/vhosts/cottos.fr/httpdocs/wp-content/plugins/elementor/includes/conditions.php::87] Undefined index: motion_fx_transform_x_anchor_point [array (
  'trace' => '
#0: /var/www/vhosts/cottos.fr/httpdocs/wp-content/plugins/elementor/includes/conditions.php(87): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /var/www/vhosts/cottos.fr/httpdocs/wp-content/plugins/elementor/includes/base/controls-stack.php(1383): Elementor\Conditions :: check()
#2: /var/www/vhosts/cottos.fr/httpdocs/wp-content/plugins/elementor/includes/base/controls-stack.php(1169): Elementor\Controls_Stack -> is_control_visible()
#3: /var/www/vhosts/cottos.fr/httpdocs/wp-content/plugins/elementor/includes/base/controls-stack.php(1210): Elementor\Controls_Stack -> get_active_settings()
#4: /var/www/vhosts/cottos.fr/httpdocs/wp-content/plugins/dynamicconditions/Public/DynamicConditionsPublic.php(134): Elementor\Controls_Stack -> get_settings_for_display()
',
)]
PHP: 2023-03-07 15:32:04 [warning X 352][/home/data1user/vhosts/cottos.fr/httpdocs/wp-content/plugins/elementor/includes/conditions.php::87] Undefined array key "motion_fx_transform_x_anchor_point" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-03-19 04:44:25 [warning X 6][/home/data1user/vhosts/cottos.fr/httpdocs/wp-content/plugins/elementor/includes/conditions.php::87] Undefined array key "motion_fx_transform_x_anchor_point" [array (
  'trace' => '
#0: /home/data1user/vhosts/cottos.fr/httpdocs/wp-content/plugins/elementor/includes/conditions.php(87): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /home/data1user/vhosts/cottos.fr/httpdocs/wp-content/plugins/elementor/includes/base/controls-stack.php(1383): Elementor\Conditions :: check()
#2: /home/data1user/vhosts/cottos.fr/httpdocs/wp-content/plugins/elementor/includes/base/controls-stack.php(1169): Elementor\Controls_Stack -> is_control_visible()
#3: /home/data1user/vhosts/cottos.fr/httpdocs/wp-content/plugins/elementor/includes/base/controls-stack.php(1210): Elementor\Controls_Stack -> get_active_settings()
#4: /home/data1user/vhosts/cottos.fr/httpdocs/wp-content/plugins/dynamicconditions/Public/DynamicConditionsPublic.php(118): Elementor\Controls_Stack -> get_settings_for_display()
',
)]
PHP: 2023-03-19 04:44:25 [warning X 6][/home/data1user/vhosts/cottos.fr/httpdocs/wp-content/plugins/elementor/includes/conditions.php::87] Undefined array key "motion_fx_transform_x_anchor_point" [array (
  'trace' => '
#0: /home/data1user/vhosts/cottos.fr/httpdocs/wp-content/plugins/elementor/includes/conditions.php(87): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /home/data1user/vhosts/cottos.fr/httpdocs/wp-content/plugins/elementor/includes/base/controls-stack.php(1383): Elementor\Conditions :: check()
#2: /home/data1user/vhosts/cottos.fr/httpdocs/wp-content/plugins/elementor/includes/base/controls-stack.php(1169): Elementor\Controls_Stack -> is_control_visible()
#3: /home/data1user/vhosts/cottos.fr/httpdocs/wp-content/plugins/elementor/includes/base/controls-stack.php(1210): Elementor\Controls_Stack -> get_active_settings()
#4: /home/data1user/vhosts/cottos.fr/httpdocs/wp-content/plugins/dynamicconditions/Public/DynamicConditionsPublic.php(134): Elementor\Controls_Stack -> get_settings_for_display()
',
)]
PHP: 2023-04-21 09:09:44 [warning X 27][/home/data1user/vhosts/cottos.fr/httpdocs/wp-content/plugins/elementor/core/common/modules/ajax/module.php::175] Undefined array key "data" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-04-24 12:19:12 [warning X 3][/home/data1user/vhosts/cottos.fr/httpdocs/wp-content/plugins/elementor/includes/managers/image.php::108] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

Log: showing 20 of 222023-03-30 09:22:08 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-03-30 09:22:08 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-03-30 09:22:08 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.7.7',
  'to' => '3.12.0',
)]
2023-03-30 09:22:08 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.7.7',
  'to' => '3.12.0',
)]
2023-03-30 09:22:08 [info] elementor::elementor_updater Started 
2023-03-30 09:22:08 [info] Elementor/Upgrades - _on_each_version Start  
2023-03-30 09:22:09 [info] Elementor/Upgrades - _on_each_version Finished 
2023-03-30 09:22:09 [info] Elementor/Upgrades - _v_3_8_0_fix_php8_image_custom_size Start  
2023-03-30 09:22:09 [info] Elementor/Upgrades - _v_3_8_0_fix_php8_image_custom_size Finished 
2023-03-30 09:22:09 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.7.8',
  'to' => '3.12.0',
)]
2023-06-09 15:05:43 [info] elementor::elementor_updater Started 
2023-06-09 15:05:43 [info] Elementor/Upgrades - _on_each_version Start  
2023-06-09 15:05:43 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.12.0',
  'to' => '3.13.4',
)]
2023-06-09 15:05:47 [info] Elementor/Upgrades - _on_each_version Finished 
2023-06-09 15:05:47 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.12.0',
  'to' => '3.13.4',
)]
2023-06-13 08:17:32 [info] elementor-pro::elementor_pro_updater Started 
2023-06-13 08:17:33 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-06-13 08:17:33 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-06-13 08:17:33 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.12.0',
  'to' => '3.13.2',
)]
2023-06-13 08:17:33 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.12.0',
  'to' => '3.13.2',
)]



== Elementor - Compatibility Tag ==
	
	Connect Polylang for Elementor: Compatibilité non spécifiée
	Elementor Pro: Compatibilité non spécifiée

== Elementor Pro - Compatibility Tag ==
	

				

````

</details>

**Comment by Guillaume-Samson** [comment]
Same problem here 👍🏻

**Comment by nicholaszein** [comment]
Hello @TelioHRV ! 

Thank you for submitting your issue! 🙏 

Please allow us some time to investigate it, and get back to you with our findings. 🧐 

Best regards

**Comment by TelioHRV** [comment]
✅ Fixed issue on elementor version 3.15.

**Comment by nicholaszein** [comment]
Hey,

Sorry for the delay in answering this issue. It passed under my radar. 


### This issue was actually completely solved with the release of Elementor Pro v3.16! 💪 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
